awscli/botocore/handlers.py

```diff
@@ -115,6 +115,7 @@ def escape_xml_payload(params, **kwargs):
 
 
 def check_for_200_error(response, **kwargs):
+    """This function has been deprecated, but is kept for backwards compatibility."""
```

Comment:
This isn't necessary for the CLIv2.  We should delete this function (assuming it's no longer called).

Reply:
I verified this function is not called in our code base and removed it.

Reply:
It seems it's being called by 3 pre-existing unit tests. For this reason, I think I'd prefer to leave it in. Let me know if you think removing the three unit tests and the deprecated function make sense.

Reply:
We should delete the tests if they're still calling this. It's a dead path and shouldn't be needed for the CLI v2 implementation.